### DISCUSSION TITLE
Unwrap Matic Duco

### DISCUSSION THREAD

**Discussion by spoeh-man** [revoxhere/duino-coin#1497] [General]
Ich habe leider ein Problem. Ich habe leider 2500 duco zu MaticDuco gewraped wie bekomme ich es hin diese wieder zu zu duco zu unwrappen. Über https://bsc.duinocoin.com/index.html brauche ich immer Matic die ich leider nicht habe. Eventuell kann jemand behilflich sein.

**Comment by Seelenkind** [comment]
Zum wechseln braucht man immer ein wenig Guthaben im jeweiligen Netzwerk. Matic kaufen und wieder zurück transformieren. 
Anders geht es leider nicht.
